app/utilities/subdomain.rb

```diff
@@ -0,0 +1,23 @@
+class UserSubdomain
+  def self.matches?(request)
+  	if ENV['USER_SUBDOMAIN']
+  		return (request.subdomain.present? && request.subdomain == ENV['USER_SUBDOMAIN'])
+  	else
+    	return true
+	end
+  end
+end
+
+class WopiSubdomain
+  def self.matches?(request)
+    if ENV['WOPI_ENABLED'] == "true"
```

Comment:
Well, if `ENV['WOPI_ENABLED'] == 'false'`, then this expression evaluates to false which is good. But if you simply put the statement `if ENV['WOPI_ENABLED']`, even in case that it would be set to `false`, it would evaluate to true, which is not something we want? Am I right?